src/wp-content/themes/twentyseventeen/inc/block-patterns.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Twenty Twenty Theme: Block Patterns
```

Comment:
```suggestion
 * Twenty Seventeen: Block Patterns
```

(or this could just say Block Patterns, since the `@package` identifies the theme, there doesn't seem to be a standard across other files)